src/lib/model/ContentItem.ts

```diff
@@ -84,6 +84,16 @@ export class ContentItem extends HalResource {
    */
   public lastModifiedDate?: string;
 
+  /**
+   * List of user Id' who are assigned to the content item
+   */
+  public assignees?: string[];
+
+  /**
+   * Timestamp representing when the assignees list was last updated in ISO 8601 format
+   */
+  public assignedDate?: string;
```

Comment:
Can you update the tests, to ensure these fields are accessible.

Reply:
Added a new test showing they're available.